htdocs/core/lib/company.lib.php

```diff
@@ -1222,21 +1222,30 @@ function show_actions_done($conf, $langs, $db, $filterobj, $objcon = '', $noprin
     $numaction = 0 ;
     $now=dol_now('tzuser');
 
+    // Open DSI -- Fix order by -- Begin
+    $sortfield_list = explode(',', $sortfield);
+    $sortfield_label_list = array('a.id' => 'id', 'a.datep' => 'dp', 'a.percent' => 'percent');
+    $sortfield_new_list = array();
+    foreach ($sortfield_list as $sortfield_value) {
+        $sortfield_new_list[] = $sortfield_label_list[trim($sortfield_value)];
+    }
+    $sortfield_new = implode(',', $sortfield_new_list);
+
     if (! empty($conf->agenda->enabled))
     {
         // Recherche histo sur actioncomm
-	 	if (is_object($objcon) && $objcon->id) {
-			$sql = "SELECT DISTINCT a.id, a.label,";
-		}
-		else
-		{
-			$sql = "SELECT a.id, a.label,";
-		}
-		$sql.= " a.datep as dp,";
+        if (is_object($objcon) && $objcon->id) {
+            $sql = "SELECT DISTINCT a.id, a.label as note,";
```

Comment:
Why aliasing the label with "note" ? This will create a confusion between note field.